Microservices/News/src/main/java/Chaligula/news/dto/News.java

```diff
@@ -0,0 +1,11 @@
+package Chaligula.news.dto;
+
+public record News(String news, Main main) {
+    
+    public record Main(String news) {
+    }
+}
+
+
+
+
```

Comment:
Bitte die benötigten Informationen im Record einbauen

```suggestion
import java.util.List;

public record News(List<Article> articles) {
    public record Article(Source source, String author, String title, String description, String url, String urlToImage, String publishedAt) {
        public record Source(String id, String name) {
        }
    }
}

```